io.c

```diff
@@ -2070,6 +2070,7 @@ io_writev(int argc, const VALUE *argv, VALUE io)
  *    Hello, World!
  *    foobar2
  *
+ *  Related: IO#read.
```

Comment:
I'm not sure I'd consider `IO#read` and `IO#write` as related methods. They're more like polar opposites. IMO the "related" section is more like "oh you might want to use these methods instead".

Reply:
My reading is 'and/or'. In Range (and elsewhere I think -- but not everywhere), we have min, max, minmax as related.

 When I was doing io_streams.rdoc, these were the two that were not in a family: {byte|character|line} IO; they sort of do arbitrary IO.